tickets/src/services/eventService.js

```diff
@@ -84,6 +84,11 @@ export default class EventService {
           'https://assets.unlock-protocol.com/nft-images/ethwaterloo/ethwaterloo-logomark.png'
       }
 
+      if (lockAddress == '0xaA23E8ceBd3817eeEC735Cd9aaF557cCB8e8407C') {
```

Comment:
Maybe switch to `===` for lint/consistency.

Reply:
og right! thanks!